overviewer_core/src/mc_id.h

```diff
@@ -293,6 +293,7 @@ enum mc_block_id {
     block_mossy_stone_brick_stairs = 11370,
     block_mossy_cobblestone_stairs = 11371,
     block_mossy_stone_brick_wall = 11372,
+    block_lantern = 11373,
```

Comment:
If you change something in the C extension, you should also increase `OVERVIEWER_EXTENSION_VERSION` in `overviewer.h`. That way we don't run into issues with people using an outdated C extension with an updated Python part.